pkg/capabilities/capabilities.go

```diff
@@ -369,7 +374,8 @@ func (c *RemoteTriggerConfig) ApplyDefaults() {
 }
 
 type CapabilityConfiguration struct {
-	DefaultConfig       *values.Map
-	RemoteTriggerConfig *RemoteTriggerConfig
-	RemoteTargetConfig  *RemoteTargetConfig
+	DefaultConfig          *values.Map
+	RemoteTriggerConfig    *RemoteTriggerConfig
+	RemoteTargetConfig     *RemoteTargetConfig
+	RemoteExecutableConfig *RemoteExecutableConfig
```

Comment:
Can we be consistent with the naming here -> RemoteExecutableConfig -> RemoteActionConfig?

Reply:
@cedric-cordenier based on corresponding core review you're ok with leaving this as is?

Reply:
I am, but please file a follow up so we can revisit this later